tools/packaging/kata-deploy/scripts/kata-deploy.sh

```diff
@@ -302,6 +343,16 @@ function install_artifacts() {
 					;;
 			esac	
 		fi
+
+		if [ -n "${INSTALLATION_PREFIX}" ]; then
+			# We could always do this sed, regardless, but I have a strong preference
+			# on not touching the configuration files unless extremelly needed
+			sed -i -e "s|${default_dest_dir}|${dest_dir}|g" "${kata_config_file}"
+
+			# Let's only adjust qemu_cmdline for the QEMUs that we build and ship ourselves
+			[[ "${shim}" =~ ^(qemu|qemu-snp|qemu-nvidia-gpu|qemu-nvidia-gpu-snp|qemu-sev|qemu-se)$ ]] && \
+				adjust_qemu_cmdline "$shim" "${kata_config_file}"
```

Comment:
Unnecessary nit: `s/$shim/${shim}`?

Reply:
I will add as part of the follow-up.

Reply:
I think this was missed. Not essential, but if there is another push it would be nice to have

Reply:
This was missed indeed, but not on purpose.  I've just re-pushed as we rather have it fixed now.
Thanks for re-catching this one (and I will leave this one for you to mark as resolved).